app/app/urls.py

```diff
@@ -652,6 +652,7 @@
     path('_administration/email/new_grant', retail.emails.new_grant, name='admin_new_grant'),
     path('_administration/email/new_grant_approved', retail.emails.new_grant_approved, name='admin_new_grant_approved'),
     path('_administration/email/new_supporter', retail.emails.new_supporter, name='admin_new_supporter'),
+    path('_administration/email/new_contributions', retail.emails.new_contributions, name='admin_new_contributions'),
```

Comment:
Isn't this supposed to replace `new_supporter` ? Can we reuse that or remove that ?

Reply:
It seems like you did this in https://github.com/gitcoinco/web/pull/9043 ?
I'm not sure though cause it's 2 diff PRs

Reply:
Could we get it as 1 cause that way it's wrapped into 1 -> where we replace new_supporter with new_contributions ?